grunttasks/eslint.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.config('eslint', {
+    options: {
+      eslintrc: '.eslintrc'
+    },
+    files: [
+      '{,bin/,config/,grunttasks/,lib/**/,test/,test/lib/mocks}*.js'
```

Comment:
You could try something like:
`'{,bin/,config/,grunttasks/,lib/**/,test/**/,!test/lib/blanket}*.js'`

Or maybe split it into two things:
```js
'{,bin/,config/,grunttasks/,lib/**/,test/**/}*.js',
'!test/lib/blanket.js'
```

Or if you're saying it's because of the `copyright` task, exclude the blanket.js file in _grunttasks/copyright.js_:
```js
      src: [
        '<%= eslint.files %>',
        '!test/lib/blanket.js'
      ]
```